tests/Feature/GroupCourseDivisionTest.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+use App\Models\Event;
+use App\Models\User;
+use App\Models\Course;
+use App\Models\Registration;
+use App\Models\Group;
+use App\Helpers\GroupCourseDivision;
+
+
+uses(\Illuminate\Foundation\Testing\RefreshDatabase::class);
+
+beforeEach(function () {
+    $this->event = Event::factory()->create(['consider_alcohol' => true])->first();
+    $this->course = Course::factory()->count(1)->create()->first();
```

Comment:
I think the course devision should be tested with more than one course.

Reply:
outdated, CourseDivisionTest was rewritten